### PR TITLE
PE-1559: remove wallets with no byte-count from output

### PR DESCRIPTION
**Pull request by matibat**

### CODE REVIEW

File: src/daily_output.ts [hunk 1]
```diff
@@ -290,13 +292,13 @@ export class DailyOutput {
 
 			const daysDiff = daysDiffInEST(previousDate, queryDate);
 			for (let index = 0; index < daysDiff; index++) {
-				console.log(`Prev block: ${previousBlockHeight}, current block: ${height}`);
+				console.log(`Prev block: ${this.data.blockHeight}, current block: ${height}`);
```
Comment (fedellen) [inline src/daily_output.ts]: What are the differences between `this.data.blockHeight`, `height`, and `previousBlockHeight` at this scope?

**Review by fedellen — DISMISSED** [review]
Looking good -- left a question about the usage of different variables tracking block height

File: src/daily_output.ts (hunk 1 above)
Comment (matibat) [reply src/daily_output.ts]: I might have to rename those, but...
- `previousBlockHeight` is the height written on the daily_output.json at the previous run
- `height` is the currently iterated height
- `this.data.blockHeight` is the block iterated before the current one

Comment (karlprieb) [reply src/daily_output.ts]: Can you rename them? It's very confusing

File: src/daily_output.ts [hunk 2]
```diff
@@ -267,9 +267,10 @@ export class DailyOutput {
 		const tags = node.tags;
 		const dataSize = +node.data.size;
 
-		const height = edge.node.block.height;
-		const previousBlockHeight = this.previousData.blockHeight;
-		if (previousBlockHeight >= height) {
+		const heightOfPreviousRun = this.previousData.blockHeight;
+		const currentHeight = edge.node.block.height;
+		const previousHeigt = this.data.blockHeight;
```
Comment (fedellen) [inline src/daily_output.ts]: typo nit: `previousHeigt ` -> `previousHeight `

File: src/daily_output.ts (hunk 1 above)
Comment (matibat) [reply src/daily_output.ts]: [39c4c21](https://github.com/ardriveapp/inferno-backend/pull/27/commits/39c4c21886000f1ad01652ffc8b74497d9edb773)

File: src/daily_output.ts (hunk 2 above)
Comment (matibat) [reply src/daily_output.ts]: [19526e0](https://github.com/ardriveapp/inferno-backend/pull/27/commits/19526e0c1a504769b34120e8770ac9e96d224a39)

**Review by fedellen — APPROVED** [review]

**Review by karlprieb — APPROVED** [review]

**Pull request merged**